strongback/src/org/strongback/hardware/Hardware.java

```diff
@@ -781,7 +781,7 @@ public static InputDevice driverStationJoystick(int port) {
          * @param port the port on the driver station that the flight stick is plugged into
          * @return the input device; never null
          */
-        public static FlightStick logitechAttack3D(int port) {
+        public static FlightStick logitechAttack3(int port) {
```

Comment:
Let's keep the renaming to a separate issue. Even though technically it is wrong, it really isn't necessary to correct by changing the name of the method, since that will break existing clients that use that method. IMO, it's far more valuable to keep compatibility with prior versions so that client code doesn't break when teams upgrade to a newer version of Strongback. 

Feel free to correct the JavaDoc (e.g., change from "Attack 3D" to "Attack 3") in this pull request.